packages/contracts-bedrock/test/L2/L2StandardBridgeInterop.t.sol

```diff
@@ -145,7 +142,7 @@ contract L2StandardBridgeInterop_LegacyToSuper_Test is L2StandardBridgeInterop_T
         _mockDeployments(address(l2OptimismMintableERC20Factory), _from, _remoteToken);
 
         // Mock the superchain factory to return address(0)
-        _mockDeployments(Predeploys.OPTIMISM_SUPERCHAIN_ERC20_FACTORY, _to, address(0));
+        _mockDeployments(address(l2OptimismSuperchainERC20Factory), _to, address(0));
```

Comment:
why did you remove the `Predeploys...` way of calling it?

Reply:
I wanted to replicate the way they use the initialized predeploys across the different test suites